readme.md

```diff
@@ -1,45 +1,75 @@
+[![Build Status](https://travis-ci.com/dusk-network/Hades252.svg?branch=master)](https://travis-ci.com/dusk-network/Hades252)
+[![codecov](https://codecov.io/gh/dusk-network/Hades252/branch/master/graph/badge.svg)](https://codecov.io/gh/dusk-network/Hades252)
+[![Repository](https://dusk-network.github.io/Hades252/repo-badge.svg)](https://github.com/dusk-network/Hades252)
+[![Documentation](https://dusk-network.github.io/Hades252/badge.svg)](https://dusk-network.github.io/Hades252/index.html)
+
 # Hades252
 
 Implementation of Hades252 over the Ristretto Scalar field.
 
 *Unstable* : No guarantees can be made regarding the API stability.
 
+## Documentation
+
+To generate the `Hades252` documentation:
+
+```sh
+make doc
+make doc-internal
+```
+
+## Use
+
+To import `Hades252`, add the following to the dependencies section of your project's `Cargo.toml`:
+
+```toml
+Hades252 = "0.2"
+```
+
+By default `Hades252` has a `width` equals to `5`.
+It's possible to use an arbitrary value, between `3` and `9`, by setting the
+environment variable `HADES252_WIDTH` to the desired number.
+
 ## Parameters
 
-- p = 2^252 + 27742317777372353535851937790883648493
+- p = `2^252 + 27742317777372353535851937790883648493`
```

Comment:
Shouldn't be `Fp` ? Since it is indeed the order of the field and so, the modulo where the `Scalar`s work.